src/main/java/com/gregtechceu/gtceu/data/recipe/configurable/RecipeRemoval.java

```diff
@@ -293,17 +333,54 @@ private static void removeVanillaBlockRecipes(Consumer<ResourceLocation> registr
+        registry.accept(new ResourceLocation("minecraft:chiseled_bookshelf"));
+        registry.accept(new ResourceLocation("minecraft:quartz_pillar"));
         registry.accept(new ResourceLocation("minecraft:sea_lantern"));
+        // TODO Add extruder/laser engraver recipes for all vanilla stones to keep parity with GT stones
+        // registry.accept(new ResourceLocation("minecraft:cracked_stone_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:mossy_cobblestone_from_moss_block"));
+        // registry.accept(new ResourceLocation("minecraft:mossy_cobblestone_from_vine"));
+        // registry.accept(new ResourceLocation("minecraft:deepslate_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:cracked_nether_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:chiseled_nether_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:polished_blackstone_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:cracked_polished_blackstone_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:quartz_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:polished_deepslate"));
+        // registry.accept(new ResourceLocation("minecraft:polished_basalt"));
+        // registry.accept(new ResourceLocation("minecraft:chiseled_polished_blackstone"));
+        // registry.accept(new ResourceLocation("minecraft:deepslate_tiles"));
+        // registry.accept(new ResourceLocation("minecraft:cracked_deepslate_tiles"));
+        // registry.accept(new ResourceLocation("minecraft:chiseled_deepslate"));
+        // registry.accept(new ResourceLocation("minecraft:cracked_deepslate_bricks"));
+        // registry.accept(new ResourceLocation("minecraft:cut_red_sandstone"));
+        // registry.accept(new ResourceLocation("minecraft:polished_basalt"));
+        // registry.accept(new ResourceLocation("minecraft:polished_blackstone"));
+        // registry.accept(new ResourceLocation("minecraft:cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:exposed_cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:weathered_cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:oxidized_cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:waxed_cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:waxed_exposed_cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:waxed_weathered_cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:waxed_oxidized_cut_copper"));
+        // registry.accept(new ResourceLocation("minecraft:end_crystal"));
+        // registry.accept(new ResourceLocation("minecraft:end_rod")); // wait for approval before uncommenting this one
+        // registry.accept(new ResourceLocation("minecraft:mud_bricks")); //no other way to obtain these rn
```

Comment:
don't all these recipes already exist?

Reply:
Some of these do yes, oops

But for others:
Cracked bricks: Forge hammering the brick (only exists for stone)
Chiseled, polished, etc. are only obtainable through stonecutting. Recipes need to be added (in a future PR possibly) for laser engraving them